style.css

```diff
@@ -0,0 +1,96 @@
+*{
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+body{
+    font-family: cursive;
+    font-weight: 600;
+    min-height: 100vh;
+    background: #c0c4cd;
+
+    display: flex;
+    justify-content: center;
+    align-items: center;
+}
+.frem{
+    width: 22rem;
+    height: 91px;
+    background: white;
+    box-shadow: 12px 11px 20px #0000000f;
+    border-radius: 21px;
+
+    display: flex;
+    justify-content: flex-start;
+    align-items: center;
+
+    padding: 0 12px;
+    position: relative;
+}
+.logo{
+    width: 62px;
+    height: 62px;
+    background: #fe8a91;
+    box-shadow: 0px 0px  20px 4px  #fd4e864f;
+    border-radius: 50%;
+
+    display: flex;
+    justify-content: center;
+    align-items: center;
+}
+.logo img{
+    width: 45px;
+    filter: invert(0.2);
+}
+.title-p{
+    position: absolute;
+    top: 3px;
+    left: 89px;
+
+    font-size: 21px;
+    color: #5a5a5a;
+}
+.size-file{
+    position: absolute;
+    right: 17px;
+    width: 15.1rem;
+
+    display: flex;
+    justify-content: space-between;
+
+}
+.size-file p{
+    font-size: 14px;
+    color: #767676;
+}
+.loading{
+    position: absolute;
+    left: 5rem;
+    top: 67px;
+
+    display: flex;
+    align-items: center;
+}
+.loading::after{
+    content: '';
+    position: absolute;
+    width: 0rem;
+    height: 13px;
+    background: linear-gradient(90deg, #fecfc8, #fe818a);
+    border-radius: 23px;
+    animation: animate 1s forwards ease-in-out;
+}
+.loading::before{
+    content: '';
+    position: absolute;
+    width: 16rem;
+    height: 6px;
+    background: #b5b5b58c;
+    border-radius: 12px;
+}
+
+@keyframes animate {
+    100%{
```

Comment:
? 100%

Reply:
@murtazshaikh how exactly this working?

Reply:
we are directly reaching to our end point by specifying 100% .